### PR TITLE
added parameter names/comments for several dozen additional foodweb parameters

### PR DESCRIPTION
**Pull request by charliestock**
This pull request includes the names and comments for several dozen additional foodweb parameters with the MOM6-style formatting.  I think these are the last of the foodweb-linked parameters.  I will sweep through the entire parameter setting section again over the next week to ensure that we have all of them in the new formatting, but I think these are ready to go.  This pull request should not change answers.

### CODE REVIEW

File: generic_tracers/generic_COBALT.F90 [hunk 1]
```diff
@@ -1196,78 +1196,206 @@ subroutine user_add_params(param_file)
...
-                   default= 0.8*0.020, scale = I_sperd ) ! s-1
-    call get_param(param_file, "generic_COBALT", "bresp_mdz",  zoo(2)%bresp,   "bresp_mdz",    units="day-1", & 
-                   default= 0.008, scale = I_sperd ) ! s-1
-    call get_param(param_file, "generic_COBALT", "bresp_lgz",  zoo(3)%bresp,   "bresp_lgz",    units="day-1", & 
-                   default= 0.0032, scale = I_sperd ) ! s-1
-    !
-    !----------------------------------------------------------------------
-    ! Partitioning of zooplankton ingestion to other compartments
-    !----------------------------------------------------------------------
+    ! Prey consumed by zooplankton is either used to produce new zooplankton biomass (anabolic matabolism), respired
```
Comment (yichengt900) [inline generic_tracers/generic_COBALT.F90]: `anabolic metabolism`

File: generic_tracers/generic_COBALT.F90 [hunk 2]
```diff
@@ -1196,78 +1196,206 @@ subroutine user_add_params(param_file)
...
+    !       to the dissolved pool must be enter labile, semi-labile or semi-refractory pools.  The defaults are:
+    !
+    !                            nitrogen        phosphorus
+    !           labile             62.5%            57.5%
+    !         semi-labile          30.0%            30.0%
+    !       semi-refractory         7.5%            12.5%
+    !           total             100.0%           100.0%
+    !
+    ! IF YOU CHOOSE TO ALTER THESE PARAMETERS, BE SURE TO ALTER THEM IN A MANNER THAT ENSURES THAT ALL UNDIGESTED
+    ! MATERIAL IN ACCOUNTED FOR.  IF YOU CHANGE ONE, YOU MAY NEED TO CHANGE OTHERS!
```
Comment (yichengt900) [inline generic_tracers/generic_COBALT.F90]: `IS ACCOUNTED FOR`?

File: generic_tracers/generic_COBALT.F90 [hunk 3]
```diff
@@ -1196,78 +1196,206 @@ subroutine user_add_params(param_file)
...
+    ! Wheeler et al., 1997. (https://doi.org/10.1016/S0967-0637(96)00089-1)
+    ! Vidal et al., 1999. (https://doi.org/10.4319/lo.1999.44.1.0106)
+    !
+    call get_param(param_file, "generic_COBALT", "phi_det_smz", zoo(1)%phi_det, &
+                   "fraction of ingestion by small zooplankton to detritus", units="none", default=0.0)
+    call get_param(param_file, "generic_COBALT", "phi_det_mdz", zoo(2)%phi_det, &
+                   "fraction of ingestion by medium zooplankton to detritus", units="none", default=0.15)
+    call get_param(param_file, "generic_COBALT", "phi_det_lgz", zoo(3)%phi_det, &
+                   "fraction of ingestion by large zooplankton to detritus", units="none", default=0.30)
+    ! partitioning of zooplanton ingestion to labile dissolved organic material
```
Comment (yichengt900) [inline generic_tracers/generic_COBALT.F90]: `zooplankton`

**Review by yichengt900 — COMMENTED** [review]
@charliestock , all look good to me! There are only a few tiny typos.

File: generic_tracers/generic_COBALT.F90 (hunk 3 above)
Comment (charliestock) [reply generic_tracers/generic_COBALT.F90]: Thanks @yichengt900.   I'll have these corrected in a new commit shortly.

**Review by yichengt900 — APPROVED** [review]
@charliestock, thank you for the prompt update! Everything looks good now, and I approve the changes. However, I will wait a little longer to allow others time to review as well.

**Review by gabyneg — APPROVED** [review]
I think this looks great!

**Comment by charliestock** [comment]
Thanks all.  Perhaps we can merge now that we have two reviews?  Once this is in, I will add a pull request for the bug fix that addresses the issue that Xiao raised so that people can start evaluating in their regional and global configurations.  @yichengt900 @gabyneg @jessluo

**Pull request merged**